config/locales/en.yml

```diff
@@ -315,6 +315,7 @@ en:
         actual_salary: Actual salary
         full_time_equivalent_salary: Full-time equivalent salary
         job_role: Role
+        school_name: Name
```

Comment:
I saw no input from @hannahvb on the ticket after Chloe raised the question about this label. I just wanted to confirm that using "Name" was something she approved.